### ISSUE TITLE
ArgumentError exception raised when sort_translation_keys is true

### ISSUE THREAD

**Issue by Sija** [fnando/i18n-js#327]
Hi,

When `sort_translation_keys` option is set to `true`, call to `I18n::JS.filtered_translations` fails with exception: `ArgumentError: comparison of TrueClass with :yesterday failed`

**Comment by PikachuEXE** [comment]
Thanks for reporting the issue
Please provide more detail, such as all/part of your translations and message + backtrace of the error

**Comment by Sija** [comment]
Below is a backtrace for exception thrown when calling `I18n::JS.filtered_translations`:

```
ArgumentError: comparison of TrueClass with :yesterday failed
from .../bundler/gems/i18n-js-78030390e365/lib/i18n/js/utils.rb:42:in `sort'
```

**Comment by PikachuEXE** [comment]
Please provide the part of translation that involves `:yesterday` & `true`
This is an unexpected case for `deep_key_sort`, we need more examples to ensure it works

**Comment by Sija** [comment]
Odd thing is: there's no `:yesterday` found anywhere in my codebase...

**Comment by PikachuEXE** [comment]
It might come from other gems, you can try emptying your translations in the app and try again
It should happen again
Do you have some i18n related gems like `rails-i18n`?

**Comment by Sija** [comment]
Nope, no `rails-i18n` included.

**Comment by PikachuEXE** [comment]
Try `I18n.backend.send(:translations).to_s.include?("yesterday")`
Might be easier to chat on Gitter (https://gitter.im/fnando/i18n-js)

**Comment by PikachuEXE** [comment]
Should be `I18n::JS.translations.to_s.include?("yesterdaty")` 
Since simple backend need to be init, and I18n::JS does that automatically

**Comment by Sija** [comment]
I've found one in [rails_admin](https://github.com/sferik/rails_admin/blob/master/config/locales/rails_admin.en.yml), although it looks quite benign to me...

**Comment by PikachuEXE** [comment]
Don't tell me it's because the `true` is NOT quoted :S

**Comment by PikachuEXE** [comment]
Let me try to fix it on my side
But I am not sure if having an unquoted `true` as a translation key is expected

**Comment by PikachuEXE** [comment]
Should be fixed in c357d7b7d43cf636bb81b05a1b19fdff59afa992

**Comment by Sija** [comment]
Works indeed, thanks!
